lib/data/content/chapter-5.yml

```diff
@@ -5,7 +5,7 @@ intro: "There is good evidence that protected areas that are effectively managed
 content: "Over the past 10 years, management effectiveness data have been gathered from 169 countries globally, using 69 different methodologies, resulting in the Global Database on Protected Areas Management Effectiveness (GD-PAME). The database currently contains 28,668 records from 21,743 different protected areas. This corresponds to data for 9.1% of protected areas reported in the WDPA, representing 19.9% of protected area coverage. To date, only 21% of countries meet the management effectiveness target (i.e. of having at least 60% of their protected area coverage assessed) on land and 16% of countries meet the target in the ocean, according to the data held in GD-PAME."
 static_map_title: "Management effectiveness per country"
 static_map_description: "Percentage of the total coverage in protected areas that is assessed for management effectiveness per country"
-map_smallprint: "Source: UNEP-WCMC and IUCN. 2018. Protected Planet: The Global Database on Protected Areas Management Effectiveness (GD-PAME), July 2018 version, Cambridge, UK: UNEP-WCMC and IUCN."
+map_smallprint: "Source: UNEP-WCMC and IUCN. 2018. Protected Planet: The World Database on Protected Areas (WDPA), July 2018 version, Cambridge, UK: UNEP-WCMC and IUCN; UNEP-WCMC and IUCN. 2018. Protected Planet: The Global Database on Protected Areas Management Effectiveness (GD-PAME), July 2018 version, Cambridge, UK: UNEP-WCMC and IUCN."
```

Comment:
Isn't this text repeating twice here?

Reply:
Good spot!

Reply:
I know it's weird, but their sources are crazy long.

But yep, in this instance I think the text has been repeated twice in error?

Reply:
Also, it looks like the text is basically the same across chapters. Instead of hardcoding it each time I wonder if we could add this into a shared space somewhere. So in the future we would only need to change it once and not to worry about mismatches or mistakes

Reply:
Oh wait have just double checked and this is the same as the content sent through by Elise - I think they are separate sources. That is a good idea - I think there's a fair amount of repetition. I can take a look at doing that.

Reply:
Great, thanks Will! Perhaps a shared yml file might work

Reply:
What do you mean by a shared file? Ideally a want a sources file that is then pulled into the chapter files, rather than accessing the sources straight from the views...

Reply:
I was thinking about adding a file named `shared.yml` or something and then put all the shared text in there instead of each chapter. So instead of pulling from the chapter-5.yml, for example, to fetch that text, you can pull it from `shared.yml`

Reply:
Ah okay, that's what I was going to do. I was hoping for a way of defining all the sources in one place and then using those variables in the chapter yamls, so that you can still see all the text for a given figure in the same place, but can't find a way to do that, so will just do this!

Reply:
Just something to remember is that some sources alter slightly, particularly by date, across chapters.